<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creat_map</class>
 <widget class="QDialog" name="creat_map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="Message_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>1461</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>20</y>
     <width>201</width>
     <height>621</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="showlen">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>显示路径长度</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="load_map">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>加载地图</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save_map">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>保存地图</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="load_bg">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>加载背景</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear_bg">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>清除屏幕</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="back">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>800</y>
     <width>1461</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>12</number>
    </property>
    <item>
     <widget class="QPushButton" name="add_node">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>添加点</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_edge">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>添加边</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="del_node">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>删除点</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="del_edge">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>删除边</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mod_node">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>修改点</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mod_edge">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>修改边</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
